doc/CHANGELOG

```diff
@@ -169,6 +169,8 @@ KallistiOS version 2.1.0 -----------------------------------------------
 - DC  Cleaned up asic functions and corrected potential bugs [DH]
 - DC  Removed cooperative threading mode -- threading is always preemptive [DH]
 - DC  Added early user init function [PC]
+- *** Added Objective-C example testing/demonstrating C Runtime [FG]
+- *** Updated Objective-C examples to include more thorough testing of the runtime aspects of the language [Andrew Apperley == AA]
```

Comment:
This line is too long. Wrap it at 80 characters.

Reply:
@andrewapperley I got it, but take note!!